src/webpackDevServer.js

```diff
@@ -1,38 +0,0 @@
-import webpack from 'webpack'
-import WebpackDevServer from 'webpack-dev-server'
-
-const defaults = {
-  host: 'localhost',
-  port: 8081,
-  noInfo: true,
-  contentBase: './_site',
-  publicPath: '/js/'
-}
-
-const webpackDevServer = (opts) => {
-  let server
-
-  const process = (files, metalsmith, done) => {
-    // Prevent from starting webpack dev server multiple times
-    if (server) {
-      done()
-      return
-    }
-    const options = Object.assign(defaults, opts, metalsmith.webpack.devServer || {})
-    metalsmith.webpack.entry['webpack-hot-reload'] = `webpack-dev-server/client?http://${options.host}:${options.port}/`
-    const compiler = webpack(Object.assign({}, metalsmith.webpack))
-
-    server = new WebpackDevServer(compiler, options)
-
-    server.listen(options.port || 8081, options.host, function () {
-      console.log(`
-  [metalsmith-webpack-dev-server]: Running webpack dev server at http://${options.host}:${options.port}
-`)
-      done()
-    })
-  }
-
-  return process
-}
-
-export default webpackDevServer
```

Comment:
We no longer need this file?

Reply:
From my checking, I can see the file is not imported anyway in the existing code, so maybe the file has been deprecated for use.

Reply:
Further checking here- I see the file was removed as part of [this](https://github.com/holidayextras/static-site-generator/commit/79b70011d02ad8f0c44f35df0ae2bcd66c046405#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556L61-L64) PR - the PR doesn't have any detail but is named after removing the webpack dev server.